tokio-io/src/io/take.rs

```diff
@@ -1,21 +1,21 @@
 use crate::{AsyncBufRead, AsyncRead};
 use futures_core::ready;
-use pin_utils::{unsafe_pinned, unsafe_unpinned};
+use pin_project::{pin_project, project};
 use std::pin::Pin;
 use std::task::{Context, Poll};
 use std::{cmp, io};
 
-/// Stream for the [`take`](super::AsyncReadExt::take) method.
+/// Reader for the [`take`](super::AsyncReadExt::take) method.
```

Comment:
Was this naming change intentional?

Reply:
Oops, I didn't intend to include that change in this PR.